jck/runtime.api/playlist.xml

```diff
@@ -1219,10 +1219,7 @@
 		<disables>
 			<disable>
 				<comment>backlog/issues/567</comment>
-				<impl>openj9</impl>
-			</disable>
```

Comment:
Could you please explain why we need to keep the excludes on for 8? Can we not enable it for all versions?